docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -17,8 +17,7 @@ Install required dependencies:
 
    python -m venv npu-env
    npu-env\Scripts\activate
-   pip install nncf==2.12 onnx==1.16.1 optimum-intel==1.19.0
-   pip install openvino==2024.5 openvino-tokenizers==2024.5 openvino-genai==2024.5
+   pip install --upgrade --upgrade-strategy eager optimum[openvino] openvino-genai>=2024.5
```

Comment:
Unfortunately, models converted without the aforementioned components fixed to certain versions are not guaranteed to work

Reply:
Since it's targeted to `2024/5` branch it should be `openvino-genai==2024.5`, shouldn't it?

I assume we still need to specify `nncf==2.12` (CC: @dmatveev )

P.S I'd prefer not changing this part at all...